tests/Feature/MemberControllerTest.php

```diff
@@ -0,0 +1,875 @@
+            'position_id' => $newMember['position_id'],
+            'gender' => $newMember['gender']
+            ]);
+    }
+
+    public function testCreateMemberHaveAvatarLessThan10MB()
+    {
+        $stub = public_path().'/img/vnvd.jpg';
+        $name = str_random(8).'.jpg';
+        $path = public_path().'/img/test/'.$name;
+        copy($stub, $path);
+        $file  = new UploadedFile($path, $name, 'image/jpg', filesize($path), null, true) ;
+        $newMember = [
+        'name' => 'thuuu',
+        'information' => 'interu',
+        'phone_number' => '09716945',
+        'birthday' => '1996-12-10',
+        'position_id' => 3,
+        'gender' => 'female',
+        'avatar' => $file,
+        ];
+        $response = $this->json('POST', 'members/create', $newMember);
+        $this->assertEquals(200, $response->status());
+        $this->assertDatabaseHas('members', [
+            'name' => $newMember['name'],
+            'information' => $newMember['information'],
+            'phone_number' => $newMember['phone_number'],
+            'birthday' => $newMember['birthday'],
+            'position_id' => $newMember['position_id'],
+            'gender' => $newMember['gender']
+            ]);
+    }
+
+    public function testCreateMemberHaveAvatarNotImage()
+    {
+        $stub = public_path().'/img/aa.txt';
+        $name = str_random(8).'.txt';
+        $path = public_path().'/img/test/'.$name;
+        copy($stub, $path);
+        $file = new UploadedFile($path, $name, 'image/jpg', filesize($path), null, true);
+        $newMember = [
+        'name' => 'thuuu',
+        'information' => 'interu',
+        'phone_number' => '09716945',
+        'birthday' => '1996-12-10',
+        'position_id' => 3,
+        'gender' => 'female',
+        'avatar' => $file,
+        ];
+        $response = $this->json('POST', 'members/create', $newMember);
+        $this->assertEquals(422, $response->status());
+        $this->assertDatabaseMissing('members', [
+            'name' => $newMember['name'],
+            'information' => $newMember['information'],
+            'phone_number' => $newMember['phone_number'],
+            'birthday' => $newMember['birthday'],
+            'position_id' => $newMember['position_id'],
+            'gender' => $newMember['gender']
+            ]);
+    }
+
+    public function testEditMemberSuccessHaveImage()
+    {
+        $stub = public_path().'/img/vnvd.jpg';
+        $name = str_random(8).'.jpg';
+        $path = public_path().'/img/test/'.$name;
+        copy($stub, $path);
+        $file = new UploadedFile($path, $name, 'image/jpg', filesize($path), null, true);
+        $array = [
+        'id' => 1,
+        'name' => 'thuuu',
+        'information' => 'interu',
+        'phone_number' => '09716945',
+        'birthday' => '1996-12-10',
+        'position_id' => 3,
+        'gender' => 'female',
+        'avatar' => $file,
+        ];
+
+        $array1 = Factory(Member::class)->create([
```

Comment:
Variable `$array1` does not seem to be used, do you really need it?

Reply:
yep i ready need it